src/js/app.jsx

```diff
@@ -39,7 +39,8 @@ import {
     faUserTie,
     faShareAlt,
     faEnvelope,
-    faLink
+    faLink,
+    faArrowCircleLeft
```

Comment:
Would like to keep these in alphabetical order if possible

Reply:
I could not agree more.